frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form-status.tsx

```diff
@@ -138,15 +138,15 @@ const UploadingStatus: React.FC<UploadingStatusProps> = ({
               className="kv--create-upload__alert"
               isInline
               variant={AlertVariant.warning}
-              title="Please don’t close this browser tab"
+              title="Please don't close this browser tab"
             >
-              Closing it will cause the upload to fail. You may still navigate the console.
+              Closing this tab will cause the upload to fail. Please wait until it completes.
```

Comment:
The purpose of this text is to let the user know they can leave the flow and they won't lose their upload. We don't want to force them to stare at this page if they don't want to :) Maybe there's a better way to word that?